src/pynxtools_em/parsers/image_tiff_zeiss.py

```diff
@@ -0,0 +1,281 @@
+                and line not in self.tmp["flat_dict_meta"]
+            ):
+                token = [value.strip() for value in txt[idx + 1].strip().split("=")]
+                if len(token) == 1:
+                    if token[0].startswith("Time :"):
+                        if token[0].replace("Time :", ""):
+                            self.tmp["flat_dict_meta"][line] = token[0].replace(
+                                "Time :", ""
+                            )
+                    elif token[0].startswith("Date :"):
+                        if token[0].replace("Date :", ""):
+                            self.tmp["flat_dict_meta"][line] = token[0].replace(
+                                "Date :", ""
+                            )
+                    else:
+                        print(f"WARNING::Ignoring line {line} token {token} !")
+                else:
+                    tmp = [value.strip() for value in token[1].split()]
+                    if len(tmp) == 1 and tmp[0] in ["On", "Yes"]:
+                        self.tmp["flat_dict_meta"][line] = True
+                    elif len(tmp) == 1 and tmp[0] in ["Off", "No"]:
+                        self.tmp["flat_dict_meta"][line] = False
+                    elif len(tmp) == 2 and tmp[1] == "°C":
+                        self.tmp["flat_dict_meta"][line] = ureg.Quantity(
+                            tmp[0], ureg.degC
+                        )
+                    elif len(tmp) == 2 and tmp[1] == "X":
+                        self.tmp["flat_dict_meta"][line] = ureg.Quantity(tmp[0])
+                    elif len(tmp) == 3 and tmp[1] == "K" and tmp[2] == "X":
+                        self.tmp["flat_dict_meta"][line] = (
+                            ureg.Quantity(tmp[0]) * 1000.0
+                        )
+                    else:
+                        try:
+                            self.tmp["flat_dict_meta"][line] = ureg.Quantity(token[1])
+                        except UndefinedUnitError:
+                            if token[1]:
+                                self.tmp["flat_dict_meta"][line] = string_to_number(
+                                    token[1]
+                                )
+                        except tokenize.TokenError:
+                            if token[1]:
+                                self.tmp["flat_dict_meta"][line] = string_to_number(
+                                    token[1]
+                                )
+                        except ValueError:
+                            if token[1]:
+                                self.tmp["flat_dict_meta"][line] = string_to_number(
+                                    token[1]
+                                )
+                        except AttributeError:
+                            if token[1]:
+                                self.tmp["flat_dict_meta"][line] = string_to_number(
+                                    token[1]
+                                )
```

Comment:
you can just bundle the errors
```suggestion
                        except (UndefinedUnitError, tokenize.TokenError, ValueError, AttributeError):
    if token[1]:
        self.tmp["flat_dict_meta"][line] = string_to_number(token[1])
```

Reply:
Outdated, as now implemented like this based on your suggestion